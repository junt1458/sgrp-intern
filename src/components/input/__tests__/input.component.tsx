import { render } from '@testing-library/react';
import Input from '../input.component';

it('should render the component', () => {
  const { container } = render(<Input id={''} type={'text'} />);
  expect(container).toMatchSnapshot();
});
