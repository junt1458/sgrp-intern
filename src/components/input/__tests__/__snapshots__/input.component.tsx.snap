// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the component 1`] = `
<div>
  <input
    class="mx-2 rounded-lg px-4 py-2 text-black"
    id=""
    type="text"
  />
</div>
`;
